pages/index.tsx

```diff
@@ -2,75 +2,41 @@ import type { GetStaticProps, NextPage } from 'next'
 import Link from 'next/link'
 import { useRouter } from 'next/router'
 import { useState } from 'react'
+import ts from 'typescript'
```

Comment:
Seems this import was added, but is unused (a few are unused actually):

![image](https://user-images.githubusercontent.com/3630698/152237858-95dc354d-a03a-41d3-a079-d924ff7ac274.png)

Reply:
I'm gonna set my vscode to just do this for me, no more manually fixing these!